### PR TITLE
New York Times Magazine webpage

### PR DESCRIPTION
**Pull request by Gracetemitope**
This is a replicate of the https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0

The following was done in this  collaborative project branch: 
Linter setup
Header, main content and footer section of the new york times web page
Positioning and alignment were well redefined.
Appropriate commit messages and good collaboration

Live Link: https://gracetemitope.github.io/Positioning-project/

snippet

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @Gracetemitope and @Teshie 

Welcome to Microverse :tada: and congratulations on submitting your first project.
Your projects do not meet the requirements, as such, you will have to make some changes and submit for another review

**Required Changes are mandatory changes you need to make before your project can be approved while Optional changes are nice-to-haves or recommendations which would not stop your projects from getting approved.**

## REQUIRED
- [ ] You are to align the elements on your page with a combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)) and ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements#introduction))
- [ ] You missed out on some things from the original page.
  -  [ ] So many of the paragraphs are missing.
  - [ ] The video you used is not the same as the one used on the original page.
  - [ ] **Spirals in the Sky** section is missing on your page.
- [ ] The hamburger icon in your navbar should be a button
- [ ] The search icon, **SPACE & COSMOS** text, and the NY Times logo in your navbar should be links
- [ ] Your footer should look exactly like that of the original page. Please, work on it.

### Make those changes and submit for another code review.
### Please, make sure you submit the same pull request link. Do not create a new pull request!


## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi, @ and @, I am [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏👏  I appreciate your hard work in replicating nytimes.com.   Kindly fix the following issues.

[REQUIRED]

- [ ] Fix the header style

| Your Copy | Source |
| :---:  |  :---:  |
|Screen Shot 2020-11-27 at 11 08 21 PM|Screen Shot 2020-11-27 at 11 34 21 PM|


- [ ] Fix the broken links in the youtube video inside the iframe.

| Your Copy | Source |
| :---:  |  :---:  |
|Screen Shot 2020-11-27 at 11 33 10 PM|Screen Shot 2020-11-27 at 11 31 05 PM|



- [ ] Add the Editor's picks section on the rightmost side. (It uses <aside> tag)
aside article

- [ ] Fix the centered text with, it is narrower the main big image

| Your Copy | Source |
| :---:  |  :---:  |
|Screen Shot 2020-11-27 at 11 38 31 PM|Screen Shot 2020-11-27 at 11 07 47 PM|


Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Review by layzrd — APPROVED** [review]
Status: Approved 🙌 🎇
Hi @Gracetemitope and @Teshie, Congratulations 🎊 👏, you did very well on this project. You meet all the requirements.

Now, you can move ahead.

Please don't forget to merge to master. Good luck with the next projects!

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Pull request merged**
